scm/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>com.github.jpmorganchase.sandboni</groupId>
         <artifactId>sandboni-core</artifactId>
-        <version>1.1.2-SNAPSHOT</version>
+        <version>1.1.4-SNAPSHOT</version>
```

Comment:
1.1.3?

Reply:
There is 1.1.3 version:
https://search.maven.org/artifact/com.github.jpmorganchase.sandboni/sandboni-scm/1.1.3